docs/ADD_NEW_EXCHANGE.md

```diff
@@ -0,0 +1,1079 @@
+	} else {
+		sigPayload := ts + method + "/api" + path
+		hmac = crypto.GetHMAC(crypto.HashSHA256, []byte(sigPayload), []byte(f.API.Credentials.Secret))
+	}
+	headers := make(map[string]string)
+	headers["FTX-KEY"] = f.API.Credentials.Key
+	headers["FTX-SIGN"] = crypto.HexEncodeToString(hmac)
+	headers["FTX-TS"] = ts
+	headers["Content-Type"] = "application/json"
+	return f.SendPayload(context.Background(), &request.Item{
+		Method:        method,
+		Path:          ftxAPIURL + path,
+		Headers:       headers,
+		Body:          body,
+		Result:        result,
+		AuthRequest:   true,
+		Verbose:       f.Verbose,
+		HTTPDebugging: f.HTTPDebugging,
+		HTTPRecording: f.HTTPRecording,
+	})
+}
+```
+
+To test authenticated functions, you must have an account with API keys and SendAuthHTTPRequest must be implemented.
+
+HTTP Mocking framework can also be added for the exchange. For reference, please see the [HTTP mock](../testdata/http_mock) package.
+
+Create authenticated functions and test along the way similar to the functions above:
+
+https://docs.ftx.com/#get-account-information:
+
+```go
+// GetAccountInfo gets account info
+func (f *FTX) GetAccountInfo() (AccountData, error) {
+	var resp AccountData
+	return resp, f.SendAuthHTTPRequest(http.MethodGet, getAccountInfo, nil, &resp)
+}
+```
+
+Get Request params for authenticated requests are sent through url.Values{}:
+
+https://docs.ftx.com/#get-withdrawal-history:
+
+```go
+// GetTriggerOrderHistory gets trigger orders that are currently open
+func (f *FTX) GetTriggerOrderHistory(marketName string, startTime, endTime time.Time, side, orderType, limit string) (TriggerOrderHistory, error) {
+	var resp TriggerOrderHistory
+	params := url.Values{}
+	if marketName != "" {
+		params.Set("market", marketName)
+	}
+	if !startTime.IsZero() && !endTime.IsZero() {
+		params.Set("start_time", strconv.FormatInt(startTime.Unix(), 10))
+		params.Set("end_time", strconv.FormatInt(endTime.Unix(), 10))
+		if startTime.After(endTime) {
+			return resp, errors.New("startTime cannot be after endTime")
+		}
+	}
+	if side != "" {
+		params.Set("side", side)
+	}
+	if orderType != "" {
+		params.Set("type", orderType)
+	}
+	if limit != "" {
+		params.Set("limit", limit)
+	}
+	return resp, f.SendAuthHTTPRequest(http.MethodGet, getTriggerOrderHistory+params.Encode(), nil, &resp)
+}
+```
+
+https://docs.ftx.com/#place-order
+
+
+Structs for unmarshalling the data are made exactly the same way as the previous functions
+
+For efficiency, a JSON to Golang converter can be used: https://mholt.github.io/json-to-go/.
+However, great care must be taken as to the values which are autogenerated. The JSON converter tool will default to whatever type it detects, but ultimately conversions to a more useful variable type would be better. For example, price and quantity on some exchange API's provide these as strings. Internally, it would be better if they're converted to the more useful float64 var type.
```

Comment:
I think this could be added in the unauthenticated section instead of here. That way the comment `Structs for unmarshalling the data are made exactly the same way as the previous functions` makes a bit more sense and helps the user earlier